extensions/amp-next-page/amp-next-page.md

```diff
@@ -163,82 +148,173 @@ Elements which are common across multiple pages can be programmatically
 
-The following configuration will recommend two more documents for the user to
-read, and hides the header and footer elements from each child document.
+```json
+[
+  {
+    "image": "https://example.com/image1.jpg",
+    "title": "This article shows first",
+    "url": "https://example.com/article1.amp.html"
+  },
+  {
+    "image": "https://example.com/image2.jpg",
+    "title": "This article shows second",
+    "url": "https://example.com/article2.amp.html"
+  }
+]
+```
+
+### Remote Configuration
+
+For remote configuration, the destination server is required to return a JSON object that has the following structure:
+
+| Key                | Value                                                                                                                                                                |
+| ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `pages` (required) | Array of pages, exactly the same format as the in the inline configration section above.                                                                             |
+| `next` (optional)  | Optional string url pointing to the next remote to query for more pages (should abide by the same rules as the initial URL, namely implement the CORS requirements.) |
+
+#### Example Remote Configuration
+
+The following configuration returned from the server will recommend two more documents for the user to read and tell `amp-next-page` to query `https://example.com/more-pages` when the user finishes reading the provided recommendations.
 
 ```json
 {
   "pages": [
     {
       "image": "https://example.com/image1.jpg",
       "title": "This article shows first",
-      "ampUrl": "https://example.com/article1.amp.html"
+      "url": "https://example.com/article1.amp.html"
     },
     {
       "image": "https://example.com/image2.jpg",
       "title": "This article shows second",
-      "ampUrl": "https://example.com/article2.amp.html"
+      "url": "https://example.com/article2.amp.html"
     }
   ],
-  "hideSelectors": [".header", ".footer"]
+  "next": "https://example.com/more-pages"
 }
 ```
 
+## Attributes
+
+### `src`
+
+The URL of the remote endpoint that returns the JSON that will be used to
+configure this `amp-next-page` component. This must be a CORS HTTP service.
+The URL's protocol must be HTTPS.
+
+{% call callout('Important', type='caution') %} Your endpoint must implement
+the requirements specified in the CORS Requests in AMP spec. {% endcall %}
+
+The `src` attribute is required unless a config has been specified inline.
+
+### `max-pages` (optional)
+
+The maximum number of pages to load and show to the user. When this number is exceeded, the footer box will be shown with links to the remaining articles. The default is `Infinity`.
+
+### `deep-parsing` (optional)
```

Comment:
Does this attribute take a value? I'm thinking it must be `true` or `false`?

Reply:
Doesn't have to but can, updated